ssg/build_remediations.py

```diff
@@ -716,15 +658,15 @@ def get_rule_dir_remediations(dir_path, remediation_type, product=None):
     return product_results + common_results
 
 
-def expand_xccdf_subs(fix, remediation_type, remediation_functions):
+def expand_xccdf_subs(fix, remediation_type):
     """For those remediation scripts utilizing some of the internal SCAP
     Security Guide remediation functions expand the selected shell variables
     and remediation functions calls with <xccdf:sub> element
 
-    This routine translates any instance of the 'populate' function call in
+    This routine translates any instance of the 'bash-populate' function call in
     the form of:
 
-            populate variable_name
+            (bash-populate variable_name)
```

Comment:
These aren't functions anymore. And we don't have any other remediation functions anymore. You will have to rewrite this comment completely especially you will have to remove the things about the remediation functions below.